.github/workflows/dev-aws-ecr.yml

```diff
@@ -0,0 +1,69 @@
+name: Docker Stuff
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build-and-push-aws:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get tag
+      run: echo "RELEASE_VERSION=${GITHUB_REF:10}" >> $GITHUB_ENV
```

Comment:
what does the `:10` part of the `GITHUB_REF:10` var mean?

Reply:
It takes the first 10 characters of `GITHUB_REF`. I just want `vx.x.x` rather than `10chars/vx.x.x`